### ISSUE TITLE
🔧 Change the Train set and Test set

### ISSUE THREAD

**Issue by AndreaBasile97** [AndreaBasile97/EnergyConsumption#8]
The actual setting provide this division:

2 months for training set till the 20:00:00 of the last day
1 month for testing set from the 00:00::00 of the next day

Since the models used have problems in predicting last months, it is possible that changing the above setting can be good solution for this issue.
